src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -434,6 +434,20 @@ protected function fixManagersAutoMappings(array $managerConfigs, array $bundles
      */
     abstract protected function getMappingResourceExtension();
 
+    /**
+     * The class name used by the various mapping drivers.
+     *
+     * @param string $driverType
+     *
+     * @return string
+     */
+    protected function getMetadataDriverClass($driverType)
+    {
+        @trigger_error(sprintf('Not declaring the "%s" method in class "%s" is deprecated since Symfony 4.4. This method will be abstract in Symfony 5.', __METHOD__, static::class), E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error(sprintf('Not declaring the "%s" method in class "%s" is deprecated since Symfony 4.4. This method will be abstract in Symfony 5.0.', __METHOD__, static::class), E_USER_DEPRECATED);
```